src/react-compat.js

```diff
@@ -150,6 +157,16 @@ if (REACT013) {
   };
 }
 
+if (REACT155) {
+  // eslint-disable-next-line import/no-extraneous-dependencies
+  PropTypes = require('prop-types');
+  // eslint-disable-next-line import/no-extraneous-dependencies
+  createClass = require('create-react-class');
```

Comment:
`create-class` should work, but I think the signature for `prop-types` may have changed recently. I'll check.